packages/app/src/cli/services/dev/app-events/app-watcher-esbuild.ts

```diff
@@ -31,35 +32,43 @@ export class ESBuildContextManager {
   setAbortSignal(signal: AbortSignal) {
     this.signal = signal
     this.signal?.addEventListener('abort', async () => {
-      const allDispose = Object.values(this.contexts).map((context) => context.dispose())
+      const allDispose = Object.values(this.contexts).map((context) => context.map((ctxt) => ctxt.dispose()))
```

Comment:
This turns into an array of arrays, not sure if `Promise.all()` will handle that correctly. Better to use a `.flat()` at the end to flatten it into a single array of Promises.